RedHatInsights/insights-remediations-frontend
# Set up fec config

# Description

Associated Jira ticket: # [(issue)](https://issues.redhat.com/browse/RHINENG-7975)

This introduces the new fec.config file together with the ts-loader.

# How to test the PR
1. Try running the app in local env
2. Make sure all config changes are valid


# Before the change


# After the change


# Dependent work link


# Checklist:
